cabal-install/src/Distribution/Client/CmdShowBuildInfo.hs

```diff
@@ -135,12 +134,12 @@ showBuildInfoAction flags@NixStyleFlags { extraFlags = (ShowBuildInfoFlags fileO
     -- Default to silent verbosity otherwise it will pollute our json output
     verbosity = fromFlagOrDefault silent (configVerbosity configFlags)
     -- Also shut up haddock since it dumps warnings to stdout
-    flags' = flags { haddockFlags = haddockFlags { haddockVerbosity = Flag silent }
-                   , configFlags = configFlags { Cabal.configTests = Flag True
-                                               , Cabal.configBenchmarks = Flag True
-                                               }
-                   }
-    cliConfig = commandLineFlagsToProjectConfig globalFlags flags'
+    -- flags' = flags { haddockFlags = haddockFlags { haddockVerbosity = Flag silent }
+    --                , configFlags = configFlags { Cabal.configTests = Flag True
+    --                                            , Cabal.configBenchmarks = Flag True
+    --                                            }
+    --                }
+    cliConfig = commandLineFlagsToProjectConfig globalFlags flags
```

Comment:
Why is this not a problem any more?

Reply:
Verbosity is still set to `silent` by default for the aforementioned reason (E.g. pollutes output and makes piping the results harder). However, since we do have a file output flag, I am not against changing the default verbosity to `normal`.

This used to enable tests and benchmarks by default, which I dislike, as we might ignore user configuration for basically no reason. There might be even a good reason why these have been disabled. So, I prefer explicit over implicit, and I think this is the more reasonable behaviour. People can and should explicitly enable these if they want information about them. (Although I just found out that tests and benchmarks are automagically enabled iff all their dependencies are already built. Still, my point stands :D)

Reply:
Haddock verbosity might be still an issue, have to test that code-path.

Reply:
I've noticed some extra lines in test output with HADDOCK in them. Not sure if related.

Reply:
Probably, but I personally don't believe that is really an issue.